web/crux/src/app/deploy/deploy.service.ts

```diff
@@ -50,6 +51,7 @@ export default class DeployService {
     private agentService: AgentService,
     imageService: ImageService,
     private mapper: DeployMapper,
+    private repository: DeployRepository,
```

Comment:
Why do we need this?

Reply:
:( service = repository and I would like to improve the layer because mixing the gRPC and HTTP will be backfire.

Reply:
:) I can see that we need to improve this, but at the current form this repository just a boilerplate, and does not resolve anything about mixing grpc and http.

Let's discuss this on a call or something.

Reply:
Removed.